src/components/Explore/utils.ts

```diff
@@ -430,6 +435,21 @@ export const exploreMetricData: {
       },
     ],
   },
+  [ExploreMetric.WEEKLY_DEATHS_PER_100K]: {
+    title: 'Weekly reported deaths per 100k',
+    name: 'Weekly reported deaths per 100k',
+    chartId: 'weekly_reported_deaths_per_100k',
+    dataMeasure: DataMeasure.INTEGER,
+    yAxisDecimalPlaces: 0,
+    seriesList: [
+      {
+        label: 'Weekly reported deaths per 100k',
+        tooltipLabel: 'Weekly reported deaths per 100k',
+        datasetId: 'weeklyDeathsPer100k',
+        type: SeriesType.LINE,
```

Comment:
I assume we want to display this metric as a line to match up with daily deaths, but let me know if otherwise.